src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -1088,8 +1089,17 @@ export default {
 	padding: 0 !important;
 }
 
+// Overwrite NcListItem styles
 :deep(.list-item) {
+	padding-block: calc(1.5 * var(--default-grid-baseline)) !important;
```

Comment:
Why change default padding?

Reply:
Was done before user status size adjustment, to compensate spacing around it. Will check, if --list-item-height can be adjusted instead

Reply:
> `--list-item-height: 2lh;`

If we want elements inside the list item to have `line-height: 1.2` and adjust paddings (height) to their lines height, can we make an entire list item (without deep) having `line-height: 1.2`?

Then it works without overrinding styles and there is no extra padding, line height `2lh` is still correct.

Reply:
Sure, but then it will evaluate to 36px. Unless we're also decrease the avatar size to 36px, it will overflow the content

Reply:
I missed, why avatar must be 38px? Not 36px, inline with the text content.

Reply:
No reason, tbh, let me push and make screenshots with 36px 🦭